presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -338,6 +344,26 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         }
 
         Table icebergTable = transaction.table();
+        ReplaceSortOrder replaceSortOrder = transaction.replaceSortOrder();
+        SortOrder sortOrder = parseSortFields(schema, getSortOrder(tableMetadata.getProperties()));
+        List<org.apache.iceberg.SortField> icebergSortFields = sortOrder.fields();
+        List<SortField> sortFields = icebergSortFields.stream().map(SortField::fromIceberg).collect(toList());
```

Comment:
After recheck the code, I think it's more suitable to use the following statement here to obtain the sort fields:

```
List<SortField> sortFields = getSupportedSortFields(icebergTable.schema(), sortOrder);
```

What's your opinion? Any misunderstanding please tell me.